src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -251,8 +251,7 @@ static JmxProxy connect(
 
       return proxy;
     } catch (IOException | ExecutionException | TimeoutException | InstanceNotFoundException e) {
-      LOG.error("Failed to establish JMX connection to {}:{}", host, port);
-      throw new ReaperException("Failure when establishing JMX connection", e);
+      throw new ReaperException("Failure when establishing JMX connection to " + host + ":" + port, e);
```

Comment:
this is the core change of the PR: we don't need to log this exception under error, as it's caught elsewhere and logged appropriately…